src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.template

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": false
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-23931"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {}
```

Comment:
I misinterpreted the original question. You **can** use `netcoreapp` without using `Microsoft.NETCore.App` but that isn't technically relevant here. What is relevant is that libraries should almost always use `netstandard` unless they need to use platform-specific APIs or features, so our template for libraries should definitely use `netstandard`.